maven_lambda/__init__.py

```diff
@@ -267,7 +269,25 @@ def get_latest_version(versions, exclude_snapshots=False):
 def upload_s3_file(bucket_name, folder, file_name, data, content_type='text/plain'):
     folder = folder.rstrip('/')
     key = '{}/{}'.format(folder, file_name)
-    return s3.Object(bucket_name, key).put(Body=data, ContentType=content_type)
+    s3.Object(bucket_name, key).put(Body=data, ContentType=content_type)
+    invalidate_cloudfront(path=key)
```

Comment:
This will invalidate even if the object is new, but I don't think that's a big deal since invalidations are cheap.